ac1/_sources/sql/sorting.rst

```diff
@@ -87,16 +87,30 @@ be ordered by ``end_station``, then within each pair of ``start_station`` and
      10
 
 
-.. fillintheblank:: bikeshare_most_recent_start_date
+.. select bike_number, start_date
+   from trip_data
+   order by start_date desc
+   limit 10
```

Comment:
Use the same SQL style as rest of file

```
SELECT
  bike_number,
  start_date
FROM
  trip_data
ORDER BY
  start_date DESC
LIMIT 10
```

Reply:
Or maybe we can just delete the comment?

Reply:
I removed it.  I was going to include the solutions as comments, but then didn't do it consistently.